python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -1,45 +1,182 @@
-from typing import Callable
+import logging
+from typing import Callable, Iterator, TypeVar, List, Any
 
+import ray
+from ray.exceptions import RayActorError
 from ray.util.sgd.v2.worker_group import WorkerGroup
 
+T = TypeVar("T")
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_next():
+    """Fetch next item from result queue on each worker."""
+    # TODO: Implement along with sgd.report()
+    return None
+
+
+def fetch_all():
+    """Fetch all items from result queue on each worker."""
+    # TODO: Implement along with sgd.report()
+    return [None]
```

Comment:
Just want to confirm that this will fetch all the items that are still in the result queue - anything dequeued already will not be included.